components/selectors/Cargo.toml

```diff
@@ -20,7 +20,7 @@ doctest = false
 
 [features]
 gecko_like_types = []
-unstable = []
+bench = []
```

Comment:
Breaking change, right?

Reply:
Yeah, but not the first since we moved this crate in this repo and stopped caring about the version number.